content/en/docs/concepts/services-networking/service.md

```diff
@@ -187,9 +187,15 @@ An ExternalName Service is a special case of Service that does not have
 selectors and uses DNS names instead. For more information, see the
 [ExternalName](#externalname) section later in this document.
 
+### Over Capacity Endpoints
+Starting in Kubernetes 1.21, if an Endpoints resource has more than 1000
+endpoints the control plane will add an annotation
```

Comment:
How about:
```markdown
If an Endpoints resource has more than 1000 endpoints then a Kubernetes v1.21 (or later)
cluster annotates that Endpoints with `endpoints.kubernetes.io/over-capacity: warning`.
This annotation indicates that the affected Endpoints object is over capacity.
```
?

Also, does https://kubernetes.io/docs/reference/labels-annotations-taints/ need updating?

Reply:
That sounds better! I have also update the known labels with the new annotation. Thanks!